app/src/main/java/swen_anigans/mathematicfanatic/QuizResultsActivity.java

```diff
@@ -71,14 +71,8 @@ protected void onCreate(Bundle savedInstanceState) {
 
         Star[] starValues = getStars(correct,total,starGroup.getChildCount());
         setStars(starViews,starValues);
-        if(picked == null) {
-            setRewardListeners();
-        } else {
-            LinearLayout slideUp = (LinearLayout) findViewById(R.id.slideuppanel);
-            pickedTV = ((TextView) slideUp.getChildAt(0));
-            slideUp.removeAllViews();
-            formRewardSelectedLayout(slideUp,picked);
-        }
```

Comment:
it was to make sure the reward stayed selected when the user changed activities & returned
- hit "go to locker" and returned"
- viewed video (playing the video counts as an activity change so when you go back from it, the oncreate gets called again)